tests/ci/run_benchmark_build_tests.sh

```diff
@@ -66,7 +66,7 @@ function build_openssl {
 function build_boringssl {
   git clone --depth 1 https://github.com/google/boringssl.git "${scratch_folder}/boringssl"
   pushd "${scratch_folder}/boringssl"
-  echo "install_if_enabled(TARGETS decrepit EXPORT OpenSSLTargets ${INSTALL_DESTINATION_DEFAULT})" >> CMakeLists.txt
+  echo "install(TARGETS decrepit EXPORT OpenSSLTargets ${INSTALL_DESTINATION_DEFAULT})" >> CMakeLists.txt
```

Comment:
`INSTALL_DESTINATION_DEFAULT` doesn't seem to be defined in the logs, is it needed or is only needed when [`(CMAKE_VERSION VERSION_LESS "3.14")`](https://github.com/google/boringssl/blob/54c956b2e668e11c75f1ee0367f1b3a0ad28eff9/CMakeLists.txt#L750)?